pico/api/src/main/java/io/helidon/pico/api/ClassNamed.java

```diff
@@ -16,28 +16,27 @@
 
 package io.helidon.pico.api;
 
-import io.helidon.common.types.AnnotationAndValueDefault;
+import java.lang.annotation.Documented;
```

Comment:
Somehow git crossed wires with the rename of `DefaultQualifierAndValueTest` -> `QualifierAndValueDefaultTest` and this new `ClassNamed` type.  Strange!